app/controllers/api/integrations_controller.rb

```diff
@@ -4,7 +4,7 @@ class Api::IntegrationsController < Api::BaseController
   before_action :find_service
   before_action :find_proxy
   before_action :authorize
-  before_action :hide_for_apiap, only: :edit
+  before_action :hide, only: :edit
```

Comment:
I see that this 'hide' does `raise ActiveRecord::RecordNotFound`.
IMO, what we should do instead is removing this `hide` method, also the `edit`, and remove the action from `routes.rb`

Reply:
This looks complex, could we leave it for a future PR?

Reply:
Sure 👍

Reply:
But add a 'TODO' comment then plz 🙈